sqlx-core/src/any/connection/backend.rs

```diff
@@ -27,7 +27,7 @@ pub trait AnyConnectionBackend: std::any::Any + Debug + Send + 'static {
 
     /// Begin a new transaction or establish a savepoint within the active transaction.
     ///
-    /// Returns a [`Transaction`] for controlling and tracking the new transaction.
+    /// Returns a [`Transaction`][crate::transaction::Transaction] for controlling and tracking the new transaction.
```

Comment:
This is actually a copypaste error. The method doesn't actually return `Transaction`.

Reply:
Removed line 30